tests/bin_expressions.move

```diff
@@ -0,0 +1,5 @@
+module a::b {
```

Comment:
Have you checked in this file just in case or is there a bug which is not addressed in this PR?

Reply:
checked in just in case